src/app/components/ProgressBar/index.tsx

```diff
@@ -1,14 +1,29 @@
+import { FC } from 'react'
+import Box from '@mui/material/Box'
 import MuiLinearProgress, { linearProgressClasses } from '@mui/material/LinearProgress'
 import { styled } from '@mui/material/styles'
 import { COLORS } from 'styles/theme/colors'
 
-export const VerticalProgressBar = styled(MuiLinearProgress)(() => ({
+type VerticalProgressBarProps = {
+  value: number
+}
+
+const barSize = 36
+
+export const VerticalProgressBar: FC<VerticalProgressBarProps> = ({ value }) => {
+  return (
+    <Box sx={{ width: barSize, height: barSize, transform: 'rotate(-90deg)' }}>
+      <StyledVerticalProgressBar variant="determinate" value={value} />
+    </Box>
+  )
+}
+
+export const StyledVerticalProgressBar = styled(MuiLinearProgress)(() => ({
```

Comment:
Probably don't want to export this one.
(and the name implies `styled(VerticalProgressBar)`)